app/test_with_resources.bzl

```diff
@@ -1,20 +1,21 @@
-def test_with_resources(file):
-  '''
-  Genrule for test files.
+"""
+Genrule for test files.
+"""
 
-  Because each databinding library must have a unique package name and manifest, resources must be
-  imported using the proper package name when building with Bazel. This genrule alters those imports
-  in order to keep Gradle building.
-  '''
-  native.genrule(
-    name = "update_" + file[0:-3],
-    srcs = [file],
-    outs = [file[0:-3] + "_updated.kt"],
-    cmd = '''
+def test_with_resources(name):
```

Comment:
Why change to name? That’s a bit ambiguous here since it’s not actually the test name, it’s the test file path.

Reply:
Without this fix linter says:

```
app/test_with_resources.bzl:5: unnamed-macro: By convention "test_with_resources" should have a keyword argument called "name".

It is considered a macro because it calls a rule or another macro "native.genrule" on line 10. (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unnamed-macro)
```

Reply:
Ah interesting! I suppose this isn’t exactly following the convention anyway, but let’s keep it as-is.